### PR TITLE
feat: html toast component

### PR DESCRIPTION
**Pull request by BogdanDanilescu**
PR to implement `Toast` for the macros

added:
- stories
- tests
- schema
- macro structure
- js logic
- docs

**SB screenshots**
Screenshot 2024-11-21 at 16 41 10
Screenshot 2024-11-21 at 16 41 36

### CODE REVIEW

**Review by ramonmulia — APPROVED** [review]
LGTM

**Review by hamza14khan — APPROVED** [review]
Great stuff!

File: packages/html/ds/styles.css [hunk 1]
```diff
@@ -10,6 +10,9 @@
 @import 'node_modules/@govie-ds/tailwind/css/components.css';
 @import 'node_modules/@govie-ds/tailwind/css/typography.css';
 
+/* @import 'notyf/notyf.min.css'; */
```
Comment (pregno) [inline packages/html/ds/styles.css:13]: remove this comment

**Review by pregno — APPROVED** [review]

File: packages/html/ds/styles.css (hunk 1 above)
Comment (BogdanDanilescu) [reply packages/html/ds/styles.css]: fixed

**Pull request merged**